deny.toml

```diff
@@ -109,7 +109,7 @@ deny = [
   #"Nokia",
 ]
 # Lint level for licenses considered copyleft
-copyleft = "warn"
```

Comment:
Oh I snuck in small change here too!

Making this tool help make sure we don't accidentally install copyleft libraries that might force us to open source this code. 

Its 'public' so its more "source available" right now. But I've marked the copyright here as all rights reserved